openhtf/util/argv.py

```diff
@@ -17,10 +17,39 @@
 """
 
 import argparse
+import sys
 
 
 def ModuleParser():
-  return argparse.ArgumentParser(add_help=False)
+  return SysArgvConsumingParser(add_help=False, consume=False)
+
+
+class SysArgvConsumingParser(argparse.ArgumentParser):
+  """ArgumentParser variant that can consume known arguments from sys.argv.
+
+  Args (in addition to the normal argparse.ArgumentParser args):
+    consume: If True, consumes known args from sys.argv upon parsing.
+  """
+  def __init__(self, *args, **kwargs):
+    self._consume = kwargs.pop('consume')
+    super(SysArgvConsumingParser, self).__init__(*args, **kwargs)
+
+  def parse_args(self):
```

Comment:
why are you overriding this? Also, it doesn't return anything, so you're just breaking the API